### PR TITLE
Fix/4591 work around performance issues with tinymce on chrome

### PR DESCRIPTION
**Pull request by raphj**
#4591

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- Automated tests would require performance tests: not done
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [X] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/helper.js [hunk 1]
```diff
@@ -746,6 +746,14 @@ export const scrollIntoViewIfNeeded = (elementToScrollTo, fixedContainer) => {
 export const darkenColor = (c) => color(c).darken(0.15).hex()
 export const lightenColor = (c) => color(c).lighten(0.15).hex()
 
+export const htmlCodeToDocumentFragment = (htmlCode) => {
+  // NOTE - RJ - 2020-04-28 - <template> provides a convenient content property.
```
Comment (raphj) [inline frontend_lib/src/helper.js:750]:
```suggestion
  // NOTE - RJ - 2021-04-28 - <template> provides a convenient content property.
```

**Review by grignards — APPROVED** [review]

**Comment by grignards** [comment]
The fix does not seem to effect the edition of a comment (I tried with the note app and in the publication page) : ~40 seconds to open the edition dialog with a 8.3MB picture.
The same picture in the space description and note is fine.

**Comment by grignards** [comment]
Tested on branch.

**Comment by raphj** [comment]
After extensive tests, it seems that assigning the `value` property of a textarea that is displayed to a enormous text is very slow in Chrome. adding style={display:none} to this textarea makes the app display instantaneously. Issue opened there: https://github.com/tracim/tracim/issues/4611

**Pull request merged**